examples/clusterctl-templates/clusterctl-cluster.yaml

```diff
@@ -35,6 +35,7 @@ metadata:
   name: ${CLUSTER_NAME}
   namespace: ${NAMESPACE}
 spec:
+  nodeDrainTimeout: ${NODE_DRAIN_TIMEOUT}
```

Comment:
you need to default it here. otherwise you will break the backwards compatibility

Reply:
Fixed